website/source/docs/install/index.html.md

```diff
@@ -76,3 +76,17 @@ it is on your PATH or you may get an error about Consul not being found.
 ```shell
 $ consul -v
 ```
+
+## Consul UI Supported Versions
```

Comment:
I am wondering if this is the best place to up the supported versions. Maybe this could go into the faq too. What do you think?

Reply:
@i0rek Would it make more sense if this was in its own page underneath Install? Unless we created a separate page for Supported Platforms/System requirements and placed it there, it would also make sense.

We do need to work on a simple docs page that defines what users should expect when installing and managing consul. cc @mikemorris

Reply:
Thanks for putting this together @jsosulska !

Good call @i0rek ! I'd agree with that, I'd say we just move the entire chunk under the FAQ's. I'd also consider we use the FAQ question:

`##Q: What browsers does the Consul UI support?` (or similar)

Which means this is about 'browser support generally' rather than just 'IE11 support'. We will want to be clear about our browser support even after IE11 has completely gone.

Just my 2 cents here though, I think this will be a great help whichever.